cymetric/metrics.py

```diff
@@ -41,6 +41,17 @@ def name(self):
 
 
 def _genmetricclass(f, name, depends, scheme):
+    """Metric class requirements.
```

Comment:
I would say "create a new metric class with a given name, dependencies, and schema."